axelar-chains-config/info/devnet-amplifier.json

```diff
@@ -591,23 +591,23 @@
           "deployer": "0xba76c6980428A0b10CFC5d8ccb61949677A61233"
         },
         "InterchainTokenService": {
-          "salt": "ITS v2.0.1 devnet-amplifier2",
+          "salt": "ITS v2.1.0 devnet-amplifier3",
           "deployer": "0xba76c6980428A0b10CFC5d8ccb61949677A61233",
           "proxySalt": "ITS v1.0.0 devnet-amplifier2",
-          "tokenManagerDeployer": "0x4Bf2315e29CacA44227E81c7484B4AF1dc7F4ED9",
-          "interchainToken": "0x5321CB9412D5F53d95D2B5a07861Dd45CBca2671",
-          "interchainTokenDeployer": "0x07DD1cE692d6D1f9DD02A81277DC06C79FFecde0",
-          "tokenManager": "0x4760fB5613c2818C85346805266B356000f256cC",
-          "tokenHandler": "0x836bF27091C254b5c8Da85408c95ED1E4A763a43",
-          "gatewayCaller": "0xD800E54A6476919B53a385eBa9D0dc882441aEb6",
-          "implementation": "0x733Ee16Fe75988b09119D698584E1d77a9FB4453",
+          "tokenManagerDeployer": "0xBb4cfC202536841149cD66820281146771456e16",
+          "interchainToken": "0x2EAdD89810cFFA2bAB15814d8d68970A8833E97D",
```

Comment:
`interchainToken` contract was modified. It shouldn't be changed. Can you revert this to the original value and deploy with a new salt and upgrade and test?